EIPS/eip-zodiac.md

```diff
@@ -0,0 +1,176 @@
+
+## Motivation
+Currently, most DAO tools and frameworks are built as somewhat monolithic systems. Wherein account and control logic are coupled, either in the same contract or in a tightly bound system of contracts. This needlessly inhibits the future flexibility of organizations using these tools and encourages platform lock-in via extraordinarily high switching costs.
+
+By using the [Zodiac](https://github.com/gnosis/zodiac) standard to decouple account and control logic, organizations are able to:
+1. Enable flexible, module-based control of programmable accounts
+2. Easily switch between frameworks without unnecessary overhead.
+3. Enable multiple control mechanism in parallel.
+4. Enable cross-chain / cross-layer governance.
+5. Progressively decentralize their governance as their project and community matures.
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+The [Zodiac](https://github.com/gnosis/zodiac) standard consists of four key concepts Avatars, Modules, Modifiers, and Guards:
+1. **Avatars** are programmable Ethereum accounts, like the [Gnosis Safe](https://gnosis-safe.io). Avatars are the address that holds balances, owns systems, executes transaction, is referenced externally, and ultimately represents your DAO. Avatars MUST expose the `IAvatar` interface.
+2. **Modules** are contracts enabled by an Avatar that implement some control logic.
+3. **Modifiers** are contracts that sit between Modules and Avatars to modify the Module's behavior. For example, they might enforce a delay on all functions a Module attempts to execute or limit the scope of transactions that can be initiated by the module. Modifiers MUST expose the `IAvatar` interface.
+4. **Guards** are contracts that can be enabled on Modules or Modifiers and implement pre- or post-checks on each transaction that the Module or Modifier they are enabled on executes. This allows Avatars to do things like limit the scope of addresses and functions that a Module or Modifier can call or ensure a certain state is never changed by a Module or Modifier. Guards should expose the `IGuard` interface. Modules, Modifiers, and Avatars that wish to be guardable MUST inherit `Guardable`, MUST call `checkTransaction()` before triggering execution on their target, and MUST call `checkAfterExecution()` after execution is complete.
+
+```solidity
+/// @title Zodiac Avatar - A contract that manages modules that can execute transactions via this contract.
+
+pragma solidity >=0.7.0 <0.9.0;
+
+import "@gnosis.pm/safe-contracts/contracts/common/Enum.sol";
```

Comment:
I think `Enum.sol` should either be made a part of the standard or removed; since it's used in the args of the functions, I'd lean towards the former.